pkg/iac/terraform/state/backend/s3_reader.go

```diff
@@ -21,6 +22,9 @@ type S3Backend struct {
 
 func NewS3Reader(path string) (*S3Backend, error) {
 
+	envProxy := envproxy.NewEnvProxy("DCTL_S3_", "AWS_")
+	envProxy.Apply()
+
```

Comment:
I would have put it just before calling `session.Must`

Reply:
done